hstream-core/src/HStream/LogStore/Base.hs

```diff
@@ -716,11 +730,11 @@ readEntriesByCount LogHandle {..} firstKey num = do
     startEntryId = fromMaybe dumbMinEntryId firstKey
 
 -- | close log
---
--- todo:
--- what should do when call close?
--- 1. free resource
--- 2. once close, should forbid operation on this LogHandle
+-- |
+-- | todo:
+-- | what should do when call close?
+-- | 1. free resource
+-- | 2. once close, should forbid operation on this LogHandle
```

Comment:
fix this haddocks

ref: https://haskell-haddock.readthedocs.io/en/latest/markup.html